pkg/sql/parser/sql.y

```diff
@@ -15770,6 +15773,7 @@ unreserved_keyword:
 | CONTROLJOB
 | CONVERSION
 | CONVERT
+| COORDINATOR_LOCALITY
```

Comment:
[minor] We may want to also add this to the bare_label list.

Reply:
done.